### DISCUSSION TITLE
[i18n] Rewrites language path not working

### DISCUSSION THREAD

**Discussion by levu304** [vercel/next.js#31280] [Help]
Hi, i'm using v10.1.0, i tried to rewrite language path in next.config.js but it's not work. It's not a bug but can anyone help me, what is wrong with my code. Thanks

This is my code

// next-i18next.config.js
`module.exports = { i18n: { defaultLocale: "vi", locales: ["en", "vi", "jp"], localeDetection: false, }, };
`
// next.config.js
`rewrites: async () => [ { source: "/:lang{/}?", destination: "/", }, { source: "/:lang/:path*", destination: "/:path*", }, ],`

**Comment by kimgiutae** [comment]
Hi @levu304 . Are you trying to make url translation for pages ?

**Reply by levu304** [reply]
@jnthnk yes, my current project is not showing language path in URL, that's why i wanna hide language path by rewrites URL
